app/assets/javascripts/discourse/app/controllers/preferences/interface.js

```diff
@@ -116,6 +130,30 @@ export default Controller.extend({
     }
   },
 
+  @discourseComputed
+  userSelectableColorSchemes() {
+    return listColorSchemes(this.site);
+  },
+
+  @discourseComputed("userSelectableColorSchemes")
+  showColorSchemeSelector(colorSchemes) {
+    return colorSchemes && colorSchemes.length;
+  },
+
+  @discourseComputed
+  selectedColorSchemeNoneLabel() {
+    return I18n.t("user.color_schemes.default_description");
+  },
+
+  selectedColorSchemeId: computed({
```

Comment:
Why is this set up to do a no-op here?

Reply:
Yeah, no good reason. My goal was to have `selectedColorSchemeId` take the value from the session as its default without updating the source, but it wasn't done right. 

I have refactored this now, with help from @jjaffeux, it's much cleaner, the values are set in the controller's `init()`. Another nicety: the session values don't need to be transiting via the `preferences-interface` route.